src/views/examples/form-elements/index.njk

```diff
@@ -1,111 +1,180 @@
-      <input class="govuk-c-radio__input" id="radio-stacked-2" type="radio" name="radio-stacked-group" value="Phone">
-      <label class="govuk-c-radio__label" for="radio-stacked-2">Phone</label>
-    </div>
-    <div class="govuk-c-radio">
-      <input class="govuk-c-radio__input" id="radio-stacked-3" type="radio" name="radio-stacked-group" value="Text">
-      <label class="govuk-c-radio__label" for="radio-stacked-3">Text message</label>
-    </div>
-  </fieldset>
+  {% call govukFieldset({
+    legendHtml: "<h2 class=\"govuk-heading-l\">How do you want to be contacted?</h2>"
+  }) %}
+    <span id="radio-stacked"></span>
+    {{ govukRadios({
+      "idPrefix": "radio-stacked",
+      "name": "radio-stacked",
+      "items": [
+        {
+          "value": "email",
+          "text": "Email"
+        },
+        {
+          "value": "phone",
+          "text": "Phone"
+        },
+        {
+          "value": "text",
+          "text": "Text message"
+        }
+      ]
+    }) }}
+  {% endcall %}
 
-  <fieldset class="govuk-c-fieldset" id="radio-inline">
-    <div class="govuk-c-radio govuk-c-radio--inline">
-      <input class="govuk-c-radio__input" id="radio-1" type="radio" name="radio-group" value="Yes">
-      <label class="govuk-c-radio__label" for="radio-1">Yes</label>
-    </div>
-    <div class="govuk-c-radio govuk-c-radio--inline">
-      <input class="govuk-c-radio__input" id="radio-2" type="radio" name="radio-group" value="No">
-      <label class="govuk-c-radio__label" for="radio-2">No</label>
-    </div>
-  </fieldset>
+  {% call govukFieldset() %}
+    <span id="radio-inline"></span>
+    {{ govukRadios({
+      "classes": "govuk-c-radio--inline",
+      "idPrefix": "radio-inline",
+      "name": "radio-inline",
+      "items": [
+        {
+          "value": "yes",
+          "text": "Yes"
+        },
+        {
+          "value": "no",
+          "text": "No"
+        }
+      ]
+    }) }}
+  {% endcall %}
 
-  <fieldset class="govuk-c-fieldset" id="checkbox-stacked">
-    <div class="govuk-c-checkbox">
-      <input class="govuk-c-checkbox__input" id="waste-type-1" name="waste-types" type="checkbox" value="waste-animal">
-      <label class="govuk-c-checkbox__label" for="waste-type-1">Waste from animal carcasses</label>
-    </div>
-    <div class="govuk-c-checkbox">
-      <input class="govuk-c-checkbox__input" id="waste-type-2" name="waste-types" type="checkbox" value="waste-mines">
-      <label class="govuk-c-checkbox__label" for="waste-type-2">Waste from mines or quarries</label>
-    </div>
-    <div class="govuk-c-checkbox">
-      <input class="govuk-c-checkbox__input" id="waste-type-3" name="waste-types" type="checkbox" disabled="disabled" value="waste-animal">
-      <label class="govuk-c-checkbox__label" for="waste-type-3">Waste from animal carcasses</label>
-    </div>
-  </fieldset>
+  {% call govukFieldset() %}
+    <span id="checkboxes-stacked"></span>
+    {{ govukRadios({
```

Comment:
These still don't seem to be checkboxes…

Reply:
Oh I thought you meant the id :man_facepalming:

Reply:
😆